components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -90,6 +90,7 @@ internal const val STORAGE_ACCESS_DOCUMENTATION_URL =
  * @property onNeedToRequestPermissions a callback invoked when permissions
  * need to be requested. Once the request is completed, [onPermissionsResult] needs to be invoked.
  * @property onShouldShowRequestPermissionRationale a callback that allows the feature to query
+ * @property shouldShowDoNotAskAgainCheckBox optional Visibility for Do not ask again Checkbox
```

Comment:
I think this property should below line 94⬇️ 
`the ActivityCompat.shouldShowRequestPermissionRationale or the Fragment.shouldShowRequestPermissionRationale values.` as it's cutting of the sentence of `onShouldShowRequestPermissionRationale`

Reply:
Done